unit/service/service.go

```diff
@@ -136,75 +146,6 @@ func (sv *Unit) Active() unit.Activation {
 	case Stop, StopSigabrt, StopPost, StopSigkill, StopSigterm, FinalSigkill, FinalSigterm:
 		return unit.Deactivating
 	default:
```

Comment:
I also hoped for that initially, but nope - it's not, so I had to add those
![asd](https://cloud.githubusercontent.com/assets/12877905/17645459/f69cfb80-61a6-11e6-8590-e136eaf32664.png)